week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/framework/dom-creators.ts

```diff
@@ -0,0 +1,20 @@
+import {  ReturnType  } from "../type";
+
+export  enum TagNameType {
+  div = "div",
+  p = "p",
+  section = "section",
+  article = "article"
+}
```

Comment:
Jeśli masz taką sytuację że wartość jest taka sama jak klucz. To polecam użyć unii typu prostego:

```typescript
type TagNameType = "div" | "p" | "section" | "article"
```

to powinno z powodzeniem wystarczyć a będzie "lżejsze" niż `enum`. (bo rozpłynie się w powietrzu po transpilacji)